### PR TITLE
[#9366] Tidy up UI design for some system messages

### LINKED ISSUE #9366: Tidy up UI design for some system messages
There are several messages that will be displayed to user regardless of page, depending on condition:

- Unsupported browser, disabled cookies (they are separate)
  screen shot 2019-01-25 at 1 18 50 am
  They need to be displayed more properly (UI tidy up) and perhaps some rewording.

- Unknown/invalid user in any non-static page:
  screen shot 2019-01-25 at 1 21 08 am
  Which pales in comparison with the V6 counterpart:
  screen shot 2019-01-25 at 1 22 04 am
  The above screenshot, however, is only applicable to student in V6. V7 can take it one step further and make it applicable to all non-static pages.

### PR DESCRIPTION
**Pull request by rrtheonlyone**
Part of #9366 

This PR helps add some design to the current set of system messages for teammates.

### No Internet Connection: 
(Message at the top will be constantly displayed)
screenshot 2019-02-07 at 6 04 54 pm

### Cookies Disabled
(Message at the top will be constantly displayed)
screenshot 2019-02-07 at 6 11 22 pm

### Unauthorized User
screenshot 2019-02-07 at 6 12 21 pm

### Google Account not known
screenshot 2019-02-07 at 6 12 02 pm

### Unsupported browser
(alert can be dimissed)
screenshot 2019-02-07 at 6 23 25 pm

### CODE REVIEW

File: src/web/app/page.component.html [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <nav class="navbar navbar-expand-lg navbar-dark fixed-top">
   <a routerLink="/web">
-    <img src="assets/images/teammateslogo.png" alt="TEAMMATES [Logo]">
+  <img src="assets/images/teammateslogo.png" alt="TEAMMATES [Logo]">
   </a>
   <button class="navbar-toggler d-lg-none" type="button" (click)="isCollapsed = !isCollapsed" [attr.aria-expanded]="!isCollapsed" aria-label="Toggle navigation">
-    <span class="navbar-toggler-icon"></span>
+  <span class="navbar-toggler-icon"></span>
```
Comment (wkurniawan07) [inline src/web/app/page.component.html]: Try not to have this kind of unrelated changes.

File: src/web/app/page.component.html [hunk 2]
```diff
@@ -42,25 +42,67 @@
...
-    In case this web browser does not display the webpage correctly you may wish to view it in the following supported browsers:
-    <ol>
-      <li>Microsoft Internet Explorer {{ minimumVersions['IE'] }}+</li>
-      <li>Chrome {{ minimumVersions['Chrome'] }}+</li>
-      <li>Firefox {{ minimumVersions['Firefox'] }}+</li>
-      <li>Safari {{ minimumVersions['Safari'] }}+</li>
-    </ol>
-    <br>
+  <div *ngIf="isUnsupportedBrowser && !alertDismissed">
+    <ngb-alert type="info" (close)="closeAlert()">
```
Comment (wkurniawan07) [inline src/web/app/page.component.html]: Not sure if this is the best way; there is nothing special about this particular display message that it can be dismissed, but not others. Let's remove the removable feature for now.

File: src/web/app/page.component.html [hunk 3]
```diff
@@ -42,25 +42,67 @@
...
+    </div>
   </div>
-  <div *ngIf="!isValidUser">
-    Invalid user!
+  <div *ngIf="!isValidUser && (isStudent || isInstructor || isAdmin)">
+    <div class="row">
+      <div class="alert alert-light col-sm-4 offset-sm-4">
+        <img src="assets/images/angry.png" style="float: left; height: 90px; margin: 0 10px 10px 0;">
+        <h5 style="padding: 30px 0 20px 0">You are not authorized to view this page.</h5>
+        <hr/>
```
Comment (wkurniawan07) [inline src/web/app/page.component.html]: Let's not close self-closing tags like `hr` and `br`, similarly for other places.

File: src/web/app/page.component.html [hunk 4]
```diff
@@ -42,25 +42,67 @@
...
+        <p style="text-align: center">Logout and return to main page.</p>
```
Comment (wkurniawan07) [inline src/web/app/page.component.html]: This wording implies that it can be clicked.

File: src/web/app/page.component.html [hunk 5]
```diff
@@ -42,25 +42,67 @@
...
   </div>
   <div *ngIf="isCookieDisabled">
-    You are currently not enabling cookies in your browser. TEAMMATES requires cookies in order to work effectively.
+    <div class="alert alert-danger" role="alert">
+      <i class="fas fa-ban"></i>&nbsp;You are currently not enabling cookies in your browser. TEAMMATES requires cookies in order to work effectively.
+    </div>
   </div>
-  <div *ngIf="!isValidUser">
-    Invalid user!
+  <div *ngIf="!isValidUser && (isStudent || isInstructor || isAdmin)">
```
Comment (wkurniawan07) [inline src/web/app/page.component.html]: You can consider this to reduce some duplications:
```html
<div *ngIf="!isValidUser">
  <div class="row">
    <div *ngIf="isStudent || isInstructor || isAdmin" class="alert alert-light col-sm-4 offset-sm-4">
      ..
    </div>
    <div *ngIf="!(isStudent || isInstructor || isAdmin)" class="alert alert-danger col-sm-8 offset-sm-2">
      ..
    </div>
  </div>
</div>
```

File: src/web/app/page.component.html (hunk 1 above)
Comment (rrtheonlyone) [reply src/web/app/page.component.html]: okay, I think my auto-prettify did that (will make sure it doesnt do that in future)

File: src/web/app/page.component.html (hunk 2 above)
Comment (rrtheonlyone) [reply src/web/app/page.component.html]: Okay, removed.

I initially thought that people who use old browsers by default may be hindered by the constant display of this message (but this is a rare case plus yeah this are better ways to do this)

File: src/web/app/page.component.html (hunk 3 above)
Comment (rrtheonlyone) [reply src/web/app/page.component.html]: Okay updated!

File: src/web/app/page.component.html (hunk 4 above)
Comment (rrtheonlyone) [reply src/web/app/page.component.html]: Actually it could be clicked in v6! However, I was not sure what link to put here.. should I simply pass in a router link to logout/home page?

File: src/web/app/page.component.html (hunk 5 above)
Comment (rrtheonlyone) [reply src/web/app/page.component.html]: Yup thats cleaner! I have refactored accordingly.
